app/src/main/java/org/fossasia/openevent/general/event/EventService.kt

```diff
@@ -23,12 +22,12 @@ class EventService(private val eventApi: EventApi, private val eventDao: EventDa
     }
 
     fun getSearchEvents(eventName: String): Flowable<List<Event>> {
-        val eventsFlowable = searchEventDao.getAllEvents()
+        val eventsFlowable = eventDao.getAllEventsByName()
```

Comment:
Search by name should always be initiated by server

Reply:
But ultimately it's the db which returns the data to the UI right? Earlier they were returned according to the startDay so I thought it would make more sense to return the events by their name

Reply:
But this will limit the search to db. We want the search to always happen on server, never on db

Reply:
Okay :+1: